tests/exec-tests.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 IMAGE_TYPE="$1"
 TEST_STEP="$2"
+# This parameter prevent the trust of the repo if a marker is found, user shall run git hooks trust...
+export CI=true
```

Comment:
Can you explain why this was necessary?
We should set up new fake repos for the tests rather than special case it here and in the prod code which effectively bypasses testing the logic I believe.

Reply:
Hello, I removed this settings and changed the behaviour. 
Sorry for the solicitations, I wasn't really familiar with Github. I've fixed the last errors in the tests that I thought were due to the implementation of the new features, but which turned out to be linked to the "unset" githook command in the tests...

Reply:
Hello, I removed this settings and changed the behaviour.
Sorry for the solicitations, I wasn't really familiar with Github. I've fixed the last errors in most tests that I thought were due to the implementation of the new features, but which turned out to be linked to the "unset" githook command in the tests...

Reply:
Some tests are not working even on master like opensuse, fedore, arch

Reply:
Ah ok, I should get those fixed or removed then.